cmake/toolchain/android_wasm.toolchain.cmake

```diff
@@ -18,7 +18,7 @@ include("${ANDROID_NDK_HOME}/build/cmake/android.toolchain.cmake")
 
 set(ANDROID_WASM 1)
 
-set(WASM_TRIPLE wasm64-unknown-unknown)
+set(WASM_TRIPLE wasm64-unknown-unknown-nativeandroid)
```

Comment:
Does ` wasm64-unknown-nativeandroid` not work ?  4 part tiplets are lesser used so I would hope we could avoid that.

Reply:
Would that be parsed correctly as an environment name, even if there are only 3 parts?  Let me try...

Reply:
Ah, yes - that seems to work.  I'll change the string to make it look more "usual".

Reply:
Done.